JDK 25 not supported

Maven output: 

```
ssc@nebula:~/code/test$ mvnd impsort:check
[INFO] -----------------------------------------< test:test >-----------------------------------------
[INFO] Building test 0.0.1-SNAPSHOT
[INFO]   from pom.xml
[INFO] ---------------------------------------------------------[ jar ]----------------------------------------------------------
[INFO] 
[INFO] --- impsort:1.12.0:check (default-cli) @ dmarc-reports-viewer ---
[INFO] --------------------------------------------------------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] --------------------------------------------------------------------------------------------------------------------------
[INFO] Total time:  0.238 s
[INFO] Finished at: 2025-10-08T17:01:18+02:00
[INFO] --------------------------------------------------------------------------------------------------------------------------
[ERROR] Failed to execute goal net.revelc.code:impsort-maven-plugin:1.12.0:check (default-cli) on project dmarc-reports-viewer: Execution default-cli of goal net.revelc.code:impsort-maven-plugin:1.12.0:check failed: No enum constant com.github.javaparser.ParserConfiguration.LanguageLevel.JAVA_25 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the '-e' switch
[ERROR] Re-run Maven using the '-X' switch to enable verbose output
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

--- a/pom.xml
+++ b/pom.xml
@@ -47,7 +47,7 @@
     <formatter.configFile>src/tools/modified-google-style.xml</formatter.configFile>
     <maven.compiler.release>17</maven.compiler.release>
     <project.build.outputTimestamp>2026-01-10T19:11:06Z</project.build.outputTimestamp>
-    <version.javaparser>3.28.0</version.javaparser>
+    <version.javaparser>3.28.1</version.javaparser>
   </properties>
   <dependencyManagement>
     <dependencies>

